src/Persistence/Initialization/VersionSeasonSix/ChaosMixes.cs

```diff
@@ -1230,27 +1223,16 @@ private ItemCrafting Level380OptionCrafting()
         randomItem7To9.RequiredItemOptions.Add(this.GameConfiguration.ItemOptionTypes.First(o => o == ItemOptionTypes.Option));
         craftingSettings.RequiredItems.Add(randomItem7To9);
 
-        var randomItem10To13 = this.Context.CreateNew<ItemCraftingRequiredItem>();
-        randomItem10To13.MaximumAmount = 1;
-        randomItem10To13.MinimumItemLevel = 10;
-        randomItem10To13.MaximumItemLevel = 13;
-        randomItem10To13.AddPercentage = 70;
-        randomItem10To13.FailResult = MixResult.StaysAsIs;
-        randomItem10To13.SuccessResult = MixResult.StaysAsIs;
-        randomItem10To13.Reference = GuardianOptionCrafting.ItemReference;
-        randomItem10To13.RequiredItemOptions.Add(this.GameConfiguration.ItemOptionTypes.First(o => o == ItemOptionTypes.Option));
-        craftingSettings.RequiredItems.Add(randomItem10To13);
-
-        var randomItem14To15 = this.Context.CreateNew<ItemCraftingRequiredItem>();
-        randomItem14To15.MaximumAmount = 1;
-        randomItem14To15.MinimumItemLevel = 14;
-        randomItem14To15.MaximumItemLevel = 15;
-        randomItem14To15.AddPercentage = 80;
-        randomItem14To15.FailResult = MixResult.StaysAsIs;
-        randomItem14To15.SuccessResult = MixResult.StaysAsIs;
-        randomItem14To15.Reference = GuardianOptionCrafting.ItemReference;
-        randomItem14To15.RequiredItemOptions.Add(this.GameConfiguration.ItemOptionTypes.First(o => o == ItemOptionTypes.Option));
-        craftingSettings.RequiredItems.Add(randomItem14To15);
+        var randomItem10To15 = this.Context.CreateNew<ItemCraftingRequiredItem>();
+        randomItem10To15.MaximumAmount = 1;
+        randomItem10To15.MinimumItemLevel = 10;
+        randomItem10To15.MaximumItemLevel = 15;
+        randomItem10To15.AddPercentage = 70;
+        randomItem10To15.FailResult = MixResult.StaysAsIs;
+        randomItem10To15.SuccessResult = MixResult.StaysAsIs;
+        randomItem10To15.Reference = GuardianOptionCrafting.ItemReference;
+        randomItem10To15.RequiredItemOptions.Add(this.GameConfiguration.ItemOptionTypes.First(o => o == ItemOptionTypes.Option));
+        craftingSettings.RequiredItems.Add(randomItem10To15);
```

Comment:
According to client it caps at 70%:
![Screenshot from 2024-11-26 16-25-18](https://github.com/user-attachments/assets/6b5c93f4-d49f-4862-bf77-73430fe3390e) ![Screenshot from 2024-11-26 16-21-55](https://github.com/user-attachments/assets/8a636feb-c608-4bf1-9c00-4c74789392f8)